sw/device/lib/crypto/drivers/entropy_test.c

```diff
@@ -0,0 +1,18 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+#include "sw/device/lib/base/memory.h"
+#include "sw/device/lib/base/status.h"
+#include "sw/device/lib/crypto/drivers/entropy_kat.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/ottf_main.h"
+
+OTTF_DEFINE_TEST_CONFIG();
+bool test_main(void) {
```

Comment:
It could, but I'm not sure what that would give us as the return type here gets piped through the OTTF to a [`test_status_t`](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_framework/status.h;drc=5484f212d6d675c90b58d9f2516738c80f3d3501;l=61) that gets backdoor read by the testbench, and all we really care about is whether the test passed or not don't we?